syne_tune/experiments/launchers/launch_remote_local.py

```diff
@@ -18,25 +18,25 @@
 
 from sagemaker.estimator import EstimatorBase
 
-from benchmarking.commons.baselines import MethodDefinitions
-from benchmarking.commons.benchmark_definitions import RealBenchmarkDefinition
-from benchmarking.commons.hpo_main_common import (
+from syne_tune.experiments.baselines import MethodDefinitions
+from syne_tune.experiments.benchmark_definitions import RealBenchmarkDefinition
+from syne_tune.experiments.launchers.hpo_main_common import (
     extra_metadata,
     ExtraArgsType,
     ConfigDict,
     DictStrKey,
     config_from_argparse,
 )
-from benchmarking.commons.hpo_main_local import (
+from syne_tune.experiments.launchers.hpo_main_local import (
```

Comment:
It is a bit counterintuitive that benchmarking logic (for example RealBenchmarkDefinitions) is available inside `launchers`. Is it possible to decouple it further?

Reply:
It is not in `launchers`, but in `benchmark_definitions`